docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md

```diff
@@ -151,7 +151,7 @@ The following manifest describes a single-instance WordPress Deployment and Serv
 
    You should see the WordPress set up page similar to the following screenshot.
 
-   ![wordpress-init](https://github.com/kubernetes/examples/blob/master/mysql-wordpress-pd/WordPress.png)
+   [wordpress-init](https://github.com/kubernetes/examples/blob/master/mysql-wordpress-pd/WordPress.png)
```

Comment:
The link directs to a GitHub image file. It should direct to the image itself. I will update the image accordingly.

Reply:
Good catch - yeah, yes, it should point to the image itself.